pkg/controller/mizar/mizar-pod-controller.go

```diff
@@ -191,7 +207,81 @@ func (c *MizarPodController) handle(keyWithEventType KeyWithEventType) error {
 		}
 	}
 
-	klog.V(4).Infof("Handling %v %s/%s/%s for event %v", controllerForMizarPod, tenant, namespace, name, eventType)
+	//The annotations of vpc and subnet should not be added into pods
+	//which use host networking
+	if eventType == EventType_Create && !obj.Spec.HostNetwork {
+		klog.V(4).Infof("Mizar-Pod-controller: get hostIP (%s) - podIP(%s)", obj.Status.HostIP, obj.Status.PodIP)
```

Comment:
No need to add prefix "Mizar-Pod-controller:" in the log. It is by default added into log. It's unnecessarily extra burden for log writer.

Reply:
Will remove them.

Reply:
removed them.